packages/clerk-js/tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "resolveJsonModule": true,
     "sourceMap": false,
     "strict": true,
-    "target": "ES2019",
+    "target": "ES2020",
```

Comment:
❓ what issue did this change resolve?

Reply:
This aligns clerk-js with the other packages. No fix here, just an overall improvement